keel-veto/src/test/kotlin/com/netflix/spinnaker/keel/veto/ApplicationVetoTests.kt

```diff
@@ -30,14 +38,39 @@ import strikt.assertions.isFalse
 import strikt.assertions.isTrue
 
 class ApplicationVetoTests : JUnit5Minutests {
-  val appName = "keeldemo"
-  val resourceName = ResourceId("ec2:securityGroup:test:us-west-2:keeldemo-managed")
+  private val appName = "keeldemo"
+  private val friggaAppName = "test"
+  private val clusterResourceId = ResourceId("ec2:securityGroup:test:us-west-2:$appName-managed")
+
+  private val imageId = "bakery:image:$friggaAppName-$appName"
+  private val imageResourceId = ResourceId(imageId)
+  private val imageSpec = ImageSpec(
```

Comment:
if you didn't want to bring the bakery module in as a test dependency you could probably create a dummy resource that fits this criteria. I don't have strong feelings either way